sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/AclDescriptorFactory.scala

```diff
@@ -74,7 +73,11 @@ object AclDescriptorFactory {
       aclBuilder.addDeny(idx, principalMatcher)
     }
 
-    aclBuilder.setDenyCode(aclJavaAnnotation.denyCode())
+    if (aclJavaAnnotation.inheritDenyCode()) {
+      aclBuilder.setDenyCode(0)
+    } else {
+      aclBuilder.setDenyCode(aclJavaAnnotation.denyCode().value)
+    }
```

Comment:
I guess we don't need to force it to `FORBIDDEN` if used on the `Main` class and `inheritDenyCode` is set to true.

At the end, we will send `0` to proxy, but proxy will convert to `7`, correct?